src/main/java/com/google/enterprise/secmgr/ntlmssp/NtlmCrypto.java

```diff
@@ -40,7 +37,7 @@
   private static final SecureRandom prng = new SecureRandom();
 
   static {
-    Security.addProvider(new CryptixCrypto());
+    Security.addProvider(new BouncyCastleProvider());
```

Comment:
Are these two things equivalent?  Why is it OK to swap these two?

Reply:
Turned out that they were not.. While Cryptix allowed / used 56-bit DES key (without parity bits), BC needs a proper 64-bit one. Added a helper function to add parity bits. Copied the module's test files from G3 (unmodified) to ensure compatibility.